week-1/interpret/time-format.js

```diff
@@ -14,16 +14,38 @@ console.log(result);
 // For the piece of code above, read the code and then answer the following questions
 
 // a) How many variable declarations are there in this program?
+// There are 7.
 
 // b) How many function calls are there?
+// There isn't any function calls.
 
 // c) Using documentation on MDN, explain what the expression movieLength % 60 represents
 
+// This represents how many seconds are in a minute.
```

Comment:
Specifically, the line
`const remainingSeconds = movieLength % 60;`
will divide `movieLength` by `60` and take the remainder.  This remainder is the number of seconds left over.   The result is assigned as the value of `remainingSeconds`. 
For example, if `movieLength` is `65`, `movieLength % 60` will be evaluated to `5`.  This gives us the number of seconds left over, because the other 60 seconds make up 1 minute.